app/models/job.rb

```diff
@@ -64,5 +64,9 @@ class << self
     def incomplete_project_job(project_ids)
       where(project_id: project_ids).where.not(status: STATUS_COMPLETED).first
     end
+
+    def incomplete_fis_jobs
+      where.not(type: %w(TarballJob VulnerabilityJob), status: STATUS_COMPLETED)
+    end
```

Comment:
Can we use `scope` to reduce this in a single line?

Reply:
I did the scope initially and robocop complained about code complexity in enlistment controller and hence I used a method.

Reply:
Yes, its updated now using `scope`